config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
 
 # Target score for pipeline.py:
-0
+4
```

Comment:
too small (!) for a student like you. Believe in yourself as I believe in you

Reply:
Thank you for inspiring me with your belief!